lib/js/components/Headers/OverlayHeader/OverlayHeader.vue

```diff
@@ -100,13 +101,15 @@
 @import '../../../../styles/settings/spacings';
 @import '../../../../styles/settings/media-queries';
 @import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/shadows';
 @import '../../../../styles/settings/typography/tokens';
 @import '../../../../styles/mixins/flex-overflow-mask';
 
 .ds-overlayHeader {
 	align-items: center;
 	background: $color-neutral-background;
 	border-bottom: 2px solid $color-neutral-border-ghost;
+	box-shadow: $shadow-s;
```

Comment:
Now we need to remove https://github.com/bethinkpl/wnl-platform/blob/c3ebeb5abb71c3fb77b5588c4d382be21020cc17/resources/assets/js/modules/multistepProblems/components/MultistepProblemOverlayView.vue#L88
It looks bad with both applied at once:
![Screenshot 2024-07-26 at 09 18 08](https://github.com/user-attachments/assets/51d551fd-5c41-4f7a-acff-90cc03af9ec7)

Reply:
Plus https://github.com/bethinkpl/wnl-platform/blob/0a7fc91265abaf83ab5393159cce7c94c3fdf3ac/resources/assets/js/modules/assessments/components/plans/editor/AssessmentsPlanEditor.vue#L195

Reply:
https://github.com/bethinkpl/wnl-platform/pull/5102/commits/0ca3b6d214bcf3a286814f647c4b18e1b7fa0999